### PR TITLE
Fixes for various issues (PropTypes in Polygon, GoogleAPI verison, ClickableMarkers not bool)

### PR DESCRIPTION
**Pull request by artokun**

### CODE REVIEW

**Comment by artokun** [comment]
Fixes: 
https://github.com/fullstackreact/google-maps-react/issues/79
https://github.com/fullstackreact/google-maps-react/issues/104
https://github.com/fullstackreact/google-maps-react/issues/105
https://github.com/fullstackreact/google-maps-react/issues/113

**Comment by jlgeering** [comment]
fixes #120

**Comment by GreatScot** [comment]
This PR seems to reverse the fix for isomorphic apps in dist/lib/GoogleApi.js. There are a couple of other places that need fixes to permit serverside rendering that I have come across too.

**Comment by jaysanatez** [comment]
Is this going to be merged soon? I'd love to clean up my console.

File: src/index.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ export class Map extends React.Component {
           zoom: this.props.zoom,
           maxZoom: this.props.maxZoom,
           minZoom: this.props.maxZoom,
-          clickableIcons: this.props.clickableIcons,
+          clickableIcons: this.props.clickableIcons || false,
```
Comment (jaysanatez) [inline src/index.js]: probably simpler to change it to `!!this.props.clickableIcons`. The `!!` syntax coerces truthy/falsy values (`null`, `0`, `1`, a valid object) into boolean type true/false values, so they will match the expected type `bool`

**Comment by artokun** [comment]
Updated thanks @jaysanatez

**Comment by leup** [comment]
This PR has reverted the fix in `./src/index.js` for issue #110 from PR #128. Fortunately it does not affect the `./dist/index.js` file

**Pull request merged**